ComponentFramework3.0/ComponentFramework/Scene0.cpp

```diff
@@ -119,23 +341,25 @@ void Scene0::processInput(const SDL_Event &SDLEvent, float deltaTime)
 		sceneCamera->ProcessKeyboard(CAMERA::LEFT, deltaTime);
 		printf("Move left");
 		break;
-
 	case SDLK_d:
 		sceneCamera->ProcessKeyboard(CAMERA::RIGHT, deltaTime);
 		printf("Move RIGHT");
-			break;
+		break;
+	case SDLK_l:
+		placeObjects("chair.obj");
+		printf("Place Object");
+		break;
 	}
 }
 
 void Scene0::processMouseInput(const SDL_Event &SDLEvent)
 {
 	int _xPos, _yPos;
-	if (SDLEvent.type == SDL_MOUSEMOTION && SDL_GetMouseState(NULL, NULL) & SDL_BUTTON(SDL_BUTTON_LEFT))
+	if (SDLEvent.type == SDL_MOUSEMOTION && SDL_GetMouseState(NULL, NULL) & SDL_BUTTON(SDL_BUTTON_RIGHT))
```

Comment:
If you notice SDL uses constants and not raw strings as well for their SDL_BUTTON_RIGHT value.